src/code/z_actor.c

```diff
@@ -2533,40 +2536,40 @@ void func_80031C3C(ActorContext* actorCtx, GlobalContext* globalCtx) {
 }
 
 /**
- * Adds a given actor instance at the front of the actor list of the specified type.
- * Also sets the actor instance as being of that type.
+ * Adds a given actor instance at the front of the actor list of the specified category.
+ * Also sets the actor instance as being of that category.
  */
-void Actor_AddToTypeList(ActorContext* actorCtx, Actor* actorToAdd, u8 actorType) {
+void Actor_AddToCategoryList(ActorContext* actorCtx, Actor* actorToAdd, u8 actorCategory) {
```

Comment:
With the new name, we can probably drop the "list", since the function's purpose is to add the actor to the category itself, whereas adding it the list is just a byproduct.
```suggestion
void Actor_AddToCategory(ActorContext* actorCtx, Actor* actorToAdd, u8 actorCategory) {
```